cmd/nerdctl/run_windows.go

```diff
@@ -61,6 +64,17 @@ func setPlatformOptions(opts []oci.SpecOpts, cmd *cobra.Command, id string) ([]o
 		opts = append(opts, oci.WithMemoryLimit(uint64(mem64)))
 	}
 
+	hostProcess, err:= cmd.Flags().GetBool("host-process")
+	if err != nil {
+		return nil, err
+	}
+	if hostProcess {
+		opts = append(opts,
+				customopts.WithAnnotation(annotations.WindowsHostProcess, strconv.FormatBool(true)),
+				customopts.WithAnnotation("microsoft.com/hostprocess-inherit-user", strconv.FormatBool(true)),
```

Comment:
Does nerdctl not have a user flag?

Reply:
Well, the preferred behavior would probably be to inherit, and if a user was passed we skip supplying this annotation.